streams/src/test/java/org/apache/kafka/streams/kstream/SessionWindowedDeserializerTest.java

```diff
@@ -44,37 +44,44 @@ public void testSessionWindowedDeserializerConstructor() {
         assertInstanceOf(StringDeserializer.class, inner, "Inner deserializer type should be StringDeserializer");
     }
 
+    @Deprecated
```

Comment:
This test uses deprecate config `WINDOWED_INNER_CLASS_SERDE` so we should mark the test as deprecated, too, to indicate, that we can remove the whole test when the deprecated config gets remove.

Similar elsewhere.